src/scripts/html-transforms.js

```diff
@@ -1,11 +1,17 @@
+/**
+ * Takes two transforms returns a new function that takes an html document,
+ * runs the two transforms functions on it, and returns the the resulting document.
```

Comment:
This actually takes any number of transforms! I might say “takes several transforms…” instead here.